janitor/timeseries.py

```diff
@@ -87,3 +87,44 @@ def _get_missing_timestamps(
     missing_timestamps = expected_df.index.difference(df.index)
 
     return expected_df.loc[missing_timestamps]
+
+
+@pf.register_dataframe_method
+def sort_timestamps_monotonically(
+    df: pd.DataFrame,
+    direction: str = "increasing"
+) -> pd.DataFrame:
+    """
+    Sort dataframe to be monotonic.
+
+    If timestamps are monotonic,
+    this function will return the dataframe unmodified.
+    If timestamps are not monotonic,
+    then the function will sort the dataframe.
+    Example usage:
+    .. code-block:: python
```

Comment:
We might need a few blank lines here for the docs to render properly.

```suggestion
    then the function will sort the dataframe.

    Example usage:

    .. code-block:: python
```

Reply:
I am wondering if doing this would introduce a flake error, but I will give it a shot